ajax/libs/angular-gridster/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "angular-gridster",
+  "description": "An implementation of gridster-like widgets for Angular JS. This directive gives you gridster behavior",
+  "filename": "angular-gridster.min.js",
+  "keywords": [
+    "angular",
+    "widget",
+    "gridster-like",
+    "angular directives"
+  ],
+  "author": "https://github.com/ManifestWebDesign/angular-gridster/graphs/contributors",
+  "homepage": "http://manifestwebdesign.github.io/angular-gridster",
```

Comment:
Let's use `https`
https://manifestwebdesign.github.io/angular-gridster/
Using https is better because it's more security for browsing. To prevent the possibility the users been attacked during the redirection.
Thank you.